src/python/pants/backend/python/lint/mypy/rules.py

```diff
@@ -113,17 +119,13 @@ async def mypy_lint(
         ),
     )
 
-    address_references = ", ".join(sorted(tgt.address.spec for tgt in transitive_targets.closure))
     process = pex.create_process(
         python_setup=python_setup,
         subprocess_encoding_environment=subprocess_encoding_environment,
         pex_path=pex.output_filename,
         pex_args=generate_args(mypy, file_list_path=file_list_path),
         input_digest=merged_input_files,
-        description=(
-            f"Run MyPy on {pluralize(len(transitive_targets.closure), 'target')}: "
```

Comment:
This was never right. We don't actually run on the whole transitive closure because we were filtering out non-Python and non-resource targets, e.g. skipping `jvm_library` targets.

I tried finding a way to accurately calculate the number of targets and gave up because it's complicated by codegen. So, we only report the actual number of files.

Reply:
This is good anyway - users think of files, not targets.

Reply:
The only thing I don't like is not including the name of any of the files we're running against. I left it off because listing for > 5 files seems a bit ridiculous. 

Maybe we should do exactly that, though? Render the names if < 5, otherwise just the # of files?

Reply:
In the medium term, it would be great to render spanids (or URLs?) on workunits, and allow users to fetch more information about them out of band.